.github/workflows/pull_request.yml

```diff
@@ -22,8 +22,7 @@ jobs:
           python-version: 3.9
 
       - name: 🖇️ Install dependencies
-        run: sudo apt-get update && sudo apt-get install libpoppler-dev poppler-utils -y
-
+        run: sudo apt-get update && sudo apt-get install libpoppler-dev poppler-utils tesseract-ocr libtesseract-dev -y
```

Comment:
I'm not sure if `libtesseract-dev` is necessary, but it's probably not a problem

Reply:
i copied this from the docs. we can try removing it if you'd like to see if the build/tests still pass. I'm not too worried about it.